src/utils/reqwest_resume/tests/get_test.rs

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+use crate::utils::reqwest_resume::get;
+use bytes::Bytes;
+use futures::StreamExt;
+use http_range_header::parse_range_header;
+use hyper::header::{self, HeaderValue};
+use hyper::service::{make_service_fn, service_fn};
+use hyper::{Body, Request, Response, Server};
+use std::convert::Infallible;
+use std::time::Duration;
+use tokio::time::sleep;
+
+const BUFFER_LEN: usize = 4096 * 2;
+const CHUNK_LEN: usize = 4096;
+
+fn extract_range_start(value: &HeaderValue, total_len: usize) -> u64 {
+    let s = std::str::from_utf8(value.as_bytes()).unwrap();
+    let parse_ranges = parse_range_header(s).unwrap();
+    let range = parse_ranges.validate(total_len as u64).unwrap();
+    *range[0].start()
+}
+
+async fn hello(req: Request<Body>) -> Result<Response<Body>, Infallible> {
```

Comment:
Some documentation would be nice here. It looks like you only abort the connection once. How about aborting more than once?

Reply:
Also, I think you can find a better name than `hello`.